homeassistant/components/amcrest/sensor.py

```diff
@@ -43,43 +51,16 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 class AmcrestSensor(SensorEntity):
     """A sensor implementation for Amcrest IP camera."""
 
-    def __init__(self, name, device, sensor_type):
+    def __init__(self, name, device, description):
         """Initialize a sensor for Amcrest camera."""
-        self._name = f"{name} {SENSORS[sensor_type][0]}"
+        # make copy do not change original dict
+        self.entity_description = copy(description)
```

Comment:
Why copy?!

Reply:
Because in the next line I change self.entrity_description.name, which (without copy) will change SENSORS, meaning the next sensor of same type will get a mangled name.